content/id/docs/concepts/cluster-administration/federation.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Federation
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+{{< deprecationfilewarning >}}
+{{< include "federation-deprecation-warning-note.md" >}}
+{{< /deprecationfilewarning >}}
+
+Laman ini menjelaskan alasan dan cara penggunaan _federation_ untuk melakukan manajemen 
+kluster Kubernetes.
+{{% /capture %}}
+
+{{% capture body %}}
+## Kenapa _Federation_ ?
+
+_Federation_ membuat proses manajemen _multiple_ kluster menjadi lebih mudah. 
```

Comment:
nit: membuat proses manajemen untuk banyak kluster menjadi ...

atau

gunakan multipel https://kbbi.web.id/multipel, then it becomes "kluster multipel" (pasangan dari "kluster tunggal")

apply this comment for the rest